docs/local-setup/running-testnet.md

```diff
@@ -13,9 +13,14 @@ We have temporarily disabled connecting to TestNet.  This limitation may affect
 
 ## Intro
 
-This will teach you how to set up a node that syncs with the official TestNet. You can take a look at the core code found here: [https://github.com/nearprotocol/nearcore](https://github.com/nearprotocol/nearcore)
+This will teach you how to set up a node that syncs with the official testnet/betanet. You can take a look at the core code found here: [https://github.com/nearprotocol/nearcore](https://github.com/nearprotocol/nearcore)
```

Comment:
capitalization of TestNet / BetaNet should remain consistent with docs conventions that appear elsewhere.  if the conventions are inconsistent, please say something since our naming should remain consistent everywhere.

Reply:
Changed testnet/betanet to TestNet/BetaNet wherever applicable